tests/test_reverse_one_to_one_relation_crudl_operations_are_supported.py

```
@@ -0,0 +1,176 @@
+        id=amz_author_profile.id
+    ).exists()
+
+
+@pytest.mark.django_db
+def test_deleting_relation_by_patch_should_work(client: Client) -> None:
+    """Test deleting a relation by using an update request."""
+    author = models.Author.objects.create(
+        name="Some author",
+        birth_date="1990-01-01",
+    )
+    amz_author_profile = models.AmazonAuthorProfile.objects.create(
```

Ruff (0.8.2)

Local variable `amz_author_profile` is assigned to but never used

Remove assignment to unused variable `amz_author_profile`

(F841)